src/figuresClasses.ts

```diff
@@ -1,11 +1,90 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Side length must be greater than zero');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Sides cannot form a valid triangle');
+    }
+
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return area ? +area.toFixed(2) : 0;
```

Comment:
The `getArea` method for the `Triangle` class uses `toFixed(2)` to round the area to two decimal places, which is correct. However, the conditional `area ? +area.toFixed(2) : 0` is unnecessary because `Math.sqrt` will not return a negative value, and the area will not be zero unless the sides are invalid, which is already checked in the constructor.